Resources/Locale/en-US/guidebook/chemistry/effects.ftl

```diff
@@ -327,6 +327,12 @@ reagent-effect-guidebook-innoculate-zombie-infection =
         *[other] cure
     } an ongoing zombie infection, and provides immunity to future infections
 
+reagent-effect-guidebook-remove-component =
+    { $chance ->
+        [1] Removes
+        *[other] remove
+    } a component
```

Comment:
this guidebook text is super underwhelming.

Not only does the average player not even know what a component is, but it doesn't even say what components are being removed or how many. At the very least this should pass in the component names, ideally with some formatting.

i.e.

`Removes monkey accent and replacement accent`.

You could also just add a field in the reagent effect for a custom effect description, since it's so open-ended.

Reply:
Thanks for the review! I've done your former suggestion, so it looks like this:

![image](https://github.com/space-wizards/space-station-14/assets/113810873/b86d1846-ceed-4742-a34d-ee4658fe3531)

It's probably not locale-friendly but I don't think fixing that is easily possible.